### ISSUE TITLE
move speedometer to summary tab for admin dashboard

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#2748]
Move this:
https://fromthepage.com/admin/progress_metrics
to this:
https://fromthepage.com/admin

**Comment by sylvieed** [comment]
Ok, thanks. The big numbers and the table should be moved?

**Comment by benwbrum** [comment]
Actually, click through to the "1,3000000" (or whatever) "Pages Transcribed"; that'll bring up a new page with a "speedometer" and other set of metrics.  That's what we want to move to the main admin dashboard.

**Comment by sylvieed** [comment]
When I click on the Transcribed Pages number, the [/admin/progress_metrics](https://fromthepage.com/admin/progress_metrics) page comes up that has the Pages Transcribed per Hour, Contributions per Hour, as well as a table with data for past weeks.

![progress metrics](https://user-images.githubusercontent.com/35716893/151717728-e285eab4-995b-4828-9f80-acf0ff909b3f.jpg)

Is this the speedometer page? And should the table be moved along with the numbers?

**Comment by benwbrum** [comment]
Yes, that's the speedometer page.  If there's room for the table, it makes sense; otherwise just move the pages/hour and contributions/hour statistics to the beginning of the admin dashboard.

**Comment by sylvieed** [comment]
OK, thanks.

We can move these lines of the progress_metrics view that represent the pages/hour and contributions/hour statistics to before line 4 of the [admin "summary" page view](https://github.com/benwbrum/fromthepage/blob/main/app/views/admin/index.html.slim#L4).https://github.com/benwbrum/fromthepage/blob/34687e70e1db933fc11d1e83a463ebd780df52a1/app/views/admin/progress_metrics.html.slim#L6-L7

I think there's enough room for the table, so we can also move these lines that represent the table to after line 24 of the [admin "summary" page view](https://github.com/benwbrum/fromthepage/blob/main/app/views/admin/index.html.slim#L24).
https://github.com/benwbrum/fromthepage/blob/34687e70e1db933fc11d1e83a463ebd780df52a1/app/views/admin/progress_metrics.html.slim#L9-L33

We also need to move everything in the progress_metrics action in the admin controller to the end of the [index action](https://github.com/benwbrum/fromthepage/blob/main/app/controllers/admin_controller.rb#L41).
https://github.com/benwbrum/fromthepage/blob/34687e70e1db933fc11d1e83a463ebd780df52a1/app/controllers/admin_controller.rb#L205-L223

Because the speedometer page will no longer be necessary, we should remove it completely. We'll delete the following:
- the [progress_metrics view](https://github.com/benwbrum/fromthepage/blob/main/app/views/admin/progress_metrics.html.slim) file
- the [progress_metrics action](https://github.com/benwbrum/fromthepage/blob/main/app/controllers/admin_controller.rb#L204-L224) method in the admin controller
- the [progress_metrics route](https://github.com/benwbrum/fromthepage/blob/main/config/routes.rb#L29) line
